README.md

```diff
@@ -121,6 +121,21 @@ animator.start();
 animationView.cancelAnimation();
 ```
 
+You can add a color filter to the whole animation, a specific layer, or specific content within a layer:
+```java
+final ColorFilter colorFilter = new PorterDuffColorFilter(Color.RED, PorterDuff.Mode.SRC_ATOP);
```

Comment:
Do you think we should create something like `SimpleColorFilter` that extends PorterDuffColorFilter and sets the mode. I wonder if many people will get confused by the xfer mode.